modules/mobile/docs/openapi.yaml

```diff
@@ -548,6 +548,179 @@ paths:
       security:
         - Bearer: []
       summary: /v0/appointments/facility/eligibility
+  /v0/appointments/facilities/{facililty_id}/clinics/{clinic_id}/slots:
+    get:
+      description: \*EXPERIMENTAL* Accepts date range for a clinic at a va facility and returns available slots for a a direct schedule appointment.
+      operationId: slot
+      parameters:
+        - description: The start date for the range of appointments slots in ISO 8601 UTC
+            format. If not provided the start date will be considered today.
+          example: 2020-10-29T07:00:00Z
+          in: query
+          name: startDate
+          schema:
+            format: date-time
+            type: string
+        - description: The end date for the range of appointments slots in ISO 8601 UTC format.
+            If not provided the end date will be 2 months from today's date
+          example: 2021-11-29T08:00:00Z
+          in: query
+          name: endDate
+          schema:
+            format: date-time
+            type: string
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/ClinicSlots.yml
+          description: OK
+        '401':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: Authentication is possible but has failed or not yet been provided.
+        '404':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: The requested resource could not be found.
+        '408':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: The response from this API timed out.
+        '422':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: Failure in processing entity
+        '500':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: An internal API error occured.
+        '503':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: An upstream service is unavailable, or its circuit breaker
+            may have been tripped.
+        '504':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: A response from an upstream service timed out.
+      security:
+        - Bearer: []
+      summary: /v0/appointments/facilities/{facililty_id}/clinics/{clinic_id}/slots
+  /v0/appointments/facilities/{facililty_id}/clinics/{service_type}:
+    get:
+      description: \*EXPERIMENTAL* Returns all available clinics at a facility for a given type of service.
```

Comment:
Was trying to find a more official way of making it obvious that these endpoints don't exist yet but couldn't find anything. Let me know if you know of anything.

Reply:
I would add something like *Work in Progress* or *WIP* to the summary that way it's visible in the sidebar and the endpoint header